editor/project_manager.cpp

```diff
@@ -1280,10 +1278,22 @@ void ProjectManager::_load_recent_projects() {
 
 		String project = _name.get_slice("/", 1);
 		String conf = path.plus_file("project.godot");
-		int config_version = 0; // Assume 0 until we know better
```

Comment:
Why remove the comment?

Reply:
Well OK, now that it's closer to the code that overrides it, it's fine without comment indeed.

Reply:
I don't see a reason to move this here though, it duplicates the number of file access calls for no real gain. We could also drop the `config_version` member since it's not used in the end (just to the check to grey out).

Reply:
The problem is that I need to read the project name to sort the projects. Maybe the config files could be stored in structs, so they're not loaded twice?

Reply:
Nevermind, I just removed the second load and added necessary fields to the struct for fetching them earlier.